.github/ISSUE_TEMPLATE/new_lint.yml

```
@@ -0,0 +1,48 @@
+name: New lint suggestion
+description: Suggest a new Cargo lint.
+labels: ["A-lint", "S-triage"]
```

I saw that we have <https://github.com/rust-lang/cargo/issues?q=A-new-lint>. Which one do we want? 

```suggestion
labels: ["A-new-lint", "S-triage"]
```